Comanche055/RT8_OP_CODES.agc

```diff
@@ -250,7 +250,7 @@ TRANSP2		CAF	XNB1EB
 #	2.	GENERATE IN MPAC THE SIGNUM FUNCTION OF MPAC:
 #		ENTRY:		RTB
 #					SIGNMPAC
-# IN EITHER CASE, RETURN IS TO TEH NEXT INTERPRETIVE INSTRUCTION IN THE CALLING SEQUENCE.
```

Comment:
There should be an extra space between TO and THE. ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1514.jpg))